### PR TITLE
created a doc page for Matrixone

### PR DESCRIPTION
**Pull request by yurihwang**
## Description

Create a doc page for the Matrix One data integration

**Fixes** #4782 

## Type of change

- [x] 📄 This change requires a documentation update

### What is the solution?

Creating a new doc page for Matrix One data integration

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by yurihwang** [comment]
I have read the CLA Document and I hereby sign the CLA.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

File: docs/data-integrations/matrixone.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* `database`: Database name to be connected
+* `ssl`: If you want to enable SSL Security **(Boolean)**
+* `ssl_ca`: Path or URL of the Certificate Authority (CA) certificate file
+* `ssl_cert`: Path name or URL of the server public key certificate file
+* `ssl_key`: The path name or URL of the server private key file
+  
+
+## Usage
+In order to make use of this handler and connect to Matrixone in MindsDB, the following syntax can be used,
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/matrixone.mdx:32]: The code blocks should be as follows:

> \```sql
> SQL GOES HERE
>\```

**Comment by martyna-mindsdb** [comment]
@yurihwang 
Please implement the changes mentioned in the review comments above.

**Comment by martyna-mindsdb** [comment]
recheckcla

**Comment by martyna-mindsdb** [comment]
@yurihwang 
Please add the changes to this PR.

**Pull request merged**